firedrake/mesh.py

```diff
@@ -1266,6 +1294,10 @@ def cell_closure(self):
         return dmcommon.closure_ordering(swarm, vertex_numbering,
                                          cell_numbering, entity_per_cell)
 
+    @utils.cached_property
+    def entity_orientations(self):
+        return None
```

Comment:
No need for a per-instance property just do:

```
entity_orientations = None
```

That will satisfy `ABCMeta` and have the same effect.

Reply:
Ok. Thanks.